test/state/vimState.test.ts

```diff
@@ -17,10 +17,7 @@ suite('VimState', () => {
     const vimState = new VimState(vscode.window.activeTextEditor!);
     const cursorStart = new Position(0, 0);
     const cursorStop = new Position(0, 1);
-    const initialCursors = [
-      new Range(cursorStart, cursorStop),
-      new Range(cursorStart, cursorStop)
-    ];
+    const initialCursors = [new Range(cursorStart, cursorStop), new Range(cursorStart, cursorStop)];
```

Comment:
`gulp forceprettier` seems to have made a few unrelated changes, I left them in since it seemed harmless